<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charseet="utf-8" />
    <meta name="viewport" content="width=device-width, initial-scale=1" />
    <title>Phylogenetic Analysis of DHFR</title>
    <link rel="stylesheet" href="/style.css" />
  </head>

  <body>
    <div class="header">
      <div>
        <h1>
          <u>Phylogenetic Analysis of Dihydrofolate reductase</u>
        </h1>
      </div>
      <div class="logo">
        <a href="https://github.com/samihamahin/phylo_DHFR" target="_blank">
          <img
            src="https://cdn.glitch.me/77fd1bef-262b-4f82-8045-a63a12a58782%2Fgithub%20logo.png?v=1639164902484"
            width="50px"
          />
        </a>
      </div>
    </div>

    <hr />
    <div class="row">
      <div class="column">
        <h2>Background</h2>

        <h3>Dihydrofoloate Reductase Protein</h3>

        <p>
          Dihydrofoloate reductase (DHRF) is a reducing agent found in various
          different organisms. Its primary function is to use NADPH (e- donor)
          to reduce dihydrofolic acid to tetrahydrofolic acid. Tetrahydrofolic
          acid is then used to syntehsize certain purines, thymidylic acid, and
          amino acids.
        </p>

        <p>
          It is composed of eight-stranded beta sheets connected to, and
          surrounded by, four alpha helices. The main active site contains a
          conserved Proline-Tryptophan sequence that is responsible for the
          actual reduction/binding of the additional hydrogen bond upon
          reduction.
        </p>

        <p>
          The inhibition of Dihydrofoloate reductase has been associated with
          megaloblastic anemia, which comes as a result of foliate deficiency.
          Alternatively, the inhibition of DHRF can also be used as a potential
          cancer-reducing agent, since the lack of resources to produce folic
          acid could act as a cell-growth regulator in cancer cells.
        </p>
        <h3>
          Phylogenetic Analysis of Proteins
        </h3>
        <p>
          The phylogenetic analysis of proteins such as DHRF can help us
          understand the evolutionary differences of said protein between
          different organisms, seeing how different sequences for the same
          protein aligns with the nature and surrounding environmental
          conditions of the organism. In addition, it can also shed some light
          as to which what parts of the sequences are conserved between
          different organisms, meaning that we can see which parts are actually
          responsible for the production of the protein (coding sequences) or
          which parts are just "junk" (introns, fillers, etc.).
        </p>
      </div>

      <div class="column">
        <h2>Methods</h2>

        <h3>Data Acquisition</h3>

        <p>
          The species of interest was determined by using a list of model
          organisms provided by Gonzolo Benegas. Three species were chosen
          randomly from each domain and kingdom considering the differences
          between each other. The NCBI Genbank Accession ID's for each species
          were found for the DHFR protein. The package Entrez-direct was used to
          populate the fasta files that included the amino acid sequence for the
          DHFR protein for each of the species of interest.
        </p>

        <h3>
          Multiple Sequence Alignment
        </h3>

        <p>
          The software MUSCLE is used for Multiple Sequence Alignment for the
          amino acid sequences for each of the species. The method was published
          by Robert C. Edgar in 2004. This software was chosen because of it
          achieves high scores of accuracy and speed. The binary tree for
          alignment is constructed using neighbor-joining and then the Kimura
          disntace matrix (Edgar).
        </p>

        <h3>
          Sequence Logo Visualization
        </h3>

        <p>
          Sequence logos were generated with the use of visualizing the
          conservation of a position in the mutliple sequence alignment. The
          sequence logos were generated by uploading the aligned fasta files
          from MUSCLE onto the web application WebLogo. This web application
          provides the ease of generating sequence logo without having to
          install dependent packages onto which WebLogo is based on the software
          Delila created by the Schenider Lab in UC Berkeley (Crooks).
        </p>

        <h3>
          Phylogenetic Tree Construction
        </h3>
        <p>
          The package FastTree is used to generate maximum-liklelihood
          phylogenetic tree from the multiple sequence alignment generated by
          MUSCLE. FastTree is used because it is more accutate than PhyML and
          distance matrix methods because it uses the Jukes-Cantor Model. In
          order to visualize the tree from a newick format, the package
          Bio.Phylo is used to create a pyplot with the tree.
        </p>
      </div>
    </div>

    <hr />

    <h2 class="header_fig">
      Figures
    </h2>

    <div class="Figures">
      <div>
        <h3>
          A. Sequence Logo of Position 1-50
        </h3>
        <img
          src="https://cdn.glitch.me/77fd1bef-262b-4f82-8045-a63a12a58782%2Fseq1-50.png?v=1639107437803"
        />
      </div>
      <div>
        <h3>
          B. Sequence Logo of Position 51-100
        </h3>
        <img
          src="https://cdn.glitch.me/77fd1bef-262b-4f82-8045-a63a12a58782%2Fseq51-100.png?v=1639107447993"
        />
      </div>

      <div>
        <h3>C. Sequence Logo of Position 101-150</h3>

        <img
          src="https://cdn.glitch.me/77fd1bef-262b-4f82-8045-a63a12a58782%2Fseq101-150.png?v=1639107454709"
        />
      </div>

      <div>
        <h3>D. Sequence Logo of Position 151-200</h3>
        <img
          src="https://cdn.glitch.me/77fd1bef-262b-4f82-8045-a63a12a58782%2Fseq151-200.png?v=1639107465585"
        />
      </div>

      <div>
        <h3>E. Sequence Logo of Position 201-250</h3>
        <img
          src="https://cdn.glitch.me/77fd1bef-262b-4f82-8045-a63a12a58782%2Fseq201-250.png?v=1639107513291"
        />
      </div>

      <div>
        <h3>F. Sequence Logo of Position 251-300</h3>
        <img
          src="https://cdn.glitch.me/77fd1bef-262b-4f82-8045-a63a12a58782%2Fseq251-300.png?v=1639107514543"
        />
      </div>

      <div>
        <h3>
          G. Sequence Logo of Position 301-350
        </h3>

        <img
          src="https://cdn.glitch.me/77fd1bef-262b-4f82-8045-a63a12a58782%2Fseq301-350.png?v=1639107517491"
        />
      </div>
      <div>
        <h3>
          H. Sequence Logo of Position 351-400
        </h3>

        <img
          src="https://cdn.glitch.me/77fd1bef-262b-4f82-8045-a63a12a58782%2Fseq351-400.png?v=1639107523252"
        />
      </div>
      <div>
        <h3>
          I. Sequence Logo of Position 401-450
        </h3>

        <img
          src="https://cdn.glitch.me/77fd1bef-262b-4f82-8045-a63a12a58782%2Fseq401-450.png?v=1639107529019"
        />
      </div>
      <div>
        <h3>
          J. Sequence Logo of Position 451-500
        </h3>

        <img
          src="https://cdn.glitch.me/77fd1bef-262b-4f82-8045-a63a12a58782%2Fseq451-500.png?v=1639107591450"
        />
      </div>
      <div>
        <h3>
          K. Sequence Logo of Position 501-555
        </h3>

        <img
          src="https://cdn.glitch.me/77fd1bef-262b-4f82-8045-a63a12a58782%2Fseq501-555.png?v=1639107597343"
        />
      </div>
      <div>
        <h3>L. Phylogenetic Tree</h3>
        <img
          id="phyloTree"
          src="https://cdn.glitch.me/77fd1bef-262b-4f82-8045-a63a12a58782%2Fphylo_tree_annotated.png?v=1639109028628"
          width="500"
          class="center"
        />
      </div>
    </div>

    <hr />

    <h2 class="header_fig">
      Discussion
    </h2>
    <div class="row">
      <div class="column">
        <h3>
          Sequence Logos
        </h3>
        <p>
          Figures A-K are Sequence Logos of the Multiple Sequence Alignment
          across all the organisms we chose to study. For each figure, the
          x-axis represents the amino acid sequence position of DHRF and the
          y-axis represents the number of bits per amino acid represented across
          all organisms. In other words, we can say that the y-axis represents
          the frequency of a particular amino acid appearing at that location
          compared to the rest of the amino acids across all model organisms.
        </p>
        <p>
          We can see that most position shows up blank, meaning that there isn't
          a concrete amino acid that's likely to show up at that particular
          positon. This "blank" amino acid is due to the MSA trying to align
          DHRF sequences from different types of organisms into one concrete
          sequence. Evidently, there is much variation in AA sequence between
          organisms, hence a lot of areas where there are gaps. In addition to
          variation, these locations could be blank as a result of transcription
          of areas of the original DHRF nucleotide sequence that aren't coding
          sequences. These regions could be regulatory elements (e.g. promoter,
          regulators, etc.), or perhaps they could also be introns that aren't
          directly involved in the physical structure of the DHRF protein.
        </p>
        <p>
          Focusing more on the regions where there are bits present, we can see
          that some amino acid positions are more conserved than others. A
          larger letter corresponds to more organisms with that amino acid
          present in their DHRF sequence. For example, the Proline-Tryptophan
          (P-W) sequence in positions 111-112 seems to be conserved over most of
          the model organisms. This is in line with what we expect, since the
          Proline-Tryptophan sequence is crucial for the active site of the
          protein (see Background).
        </p>
        <p>
          Generally, however, there seems to be much variation in amino acid
          sequence across all of the model organisms that we have chosen.
          Different organisms have evolved independently of each other, and so
          it's reasonable to presume that there will be some degree of variation
          across most organisms. Something to also consider, however, is that
          within a set of variations of amino acids within a position in the
          sequence, those amino acids are structurally similar to each other.
          For example, in position 120 the most commonly occuring variations of
          amino acids are Aspartic Acid (D) and Glutamic Acid (E). Both Aspartic
          Acid and Glutamic Acid have negatively charged side chains, meaning
          that they functionally act the same way. If not funcitonally similar,
          then the sizes are also similar as well.
        </p>
        <p>
          To generalize our findings, the variation in amino acids among
          different model organisms may be different when looking at a sequence
          logo, but functionally (and structurally), the polypeptide sequence
          amounts to pretty much the same protein that does (almost) the same
          thing for all organisms. But then, the question begs, how do we
          determine which variation of the polypeptide sequence is the earliest
          one? In other words, which sequence is most similar to some common
          ancestor to all of the model organisms? This is where the phylogenetic
          tree can help us out.
        </p>
      </div>
      <div class="column">
        <h3>
          Phylogenetic Tree
        </h3>

        <p>
          Viruses have the most diverse sequences in terms of th DHFR protein.
          This is shown in how all three viruses chosen are in three different
          clade. One is in a clade protists, another with veterbrates, and
          another with fungi.
        </p>

        <p>
          Prokaryotes are in a separate branch of the tree as explained by the
          evolutionary differences between eukarya and prokaryotes. This can
          also be shown in the fact that prokaryotes and eukaryotes are
          different domains therefore most of prokaryotes are on a separate
          branch. One of the prokaryotes strays away from this: Pseudomonas
          florescens. This prokaryote is part of a clade with other plants.
        </p>

        <p>
          All eukarya are under one branch of the phylogenetic tree. Not only do
          most kingdoms have their own clades but most phylums are in their own
          clades.
        </p>

        <p>
          This can be shown in how fungi have their own clade and are in a
          different phylum group(ascomycota) within the domain of eukaryotes.
          This can be shown how vertebrates which are part of the phylum
          chordata is in their own separate clade than invertebrates which are
          part of the phylum arthropoda.
        </p>

        <p>
          The invertebrate Drosophila busckii along with the protist
          Hartmannella cantabrigiensis strays away from this, where it shares a
          clade with vertebrates. But most of the invertebrates(two out of
          three) are in a different clade than the vertebrates. But all of the
          animals share a clade with the subspecies of vertebrates and
          invertebrates apart from the two that stray away from this, the
          protist Hartmannella cantabrigiensis and the virus Saimiriine
          gammaherpesvirus2.
        </p>

        <p>
          Even though plants and animals are part of different kingdoms, it is
          shown that their sequences for the protein DHFR are more similar than
          the kingdom Fungi. There is a clade of all of the plants together
          including a prokaryote Pseudomonas fluorescens as mentioned above.
        </p>

        <p>
          Protists are in a separate branch than eukaryotes which shows that
          even though protists are eukaryotes, the sequence of DHFR is more
          different than the eukaryotes which all share more of a similar
          sequence. Most protists share a clade (two out of three) with the
          virus Bacteriophage sp.
        </p>

        <p>
          For the phylogenetic tree, no out group was chosen because all of the
          domains and kingdoms used are different from each other.
        </p>
      </div>
    </div>

    <hr />

    <div class="row">
      <div class="column">
        <h2>
          References
        </h2>
        <ul>
          <li>
            Crooks, Gavin E., and Steven E. Brenner. “About” WebLogo, 2004,
            https://weblogo.berkeley.edu/.
          </li>
          <li>
            Edgar, R.C. MUSCLE: a multiple sequence alignment method with
            reduced time and space complexity. BMC Bioinformatics 5, 113 (2004).
            https://doi.org/10.1186/1471-2105-5-113
          </li>

          <li>
            “List of Model Organisms.” Wikipedia, Wikimedia Foundation, 16 Sept.
            2021, https://en.wikipedia.org/wiki/List_of_model_organisms.
          </li>
          <li>
            Matthews DA, Alden RA, Bolin JT, Freer ST, Hamlin R, Xuong N, Kraut
            J, Poe M, Williams M, Hoogsteen K (July 1977). "Dihydrofolate
            reductase: x-ray structure of the binary complex with methotrexate".
            Science. 197 (4302): 452–5. Bibcode:1977Sci...197..452M.
            doi:10.1126/science.17920. PMID 17920.
            https://www.ncbi.nlm.nih.gov/gene?Db=gene&Cmd=ShowDetailView&TermToSearch=1719
          </li>
        </ul>
      </div>
    </div>
  </body>
</html>
